tensorflow/core/platform/hadoop/hadoop_file_system.cc

```diff
@@ -164,7 +164,7 @@ Status HadoopFileSystem::Connect(StringPiece fname, hdfsFS* fs) {
     // https://github.com/tensorflow/tensorflow/blob/v1.0.0/third_party/hadoop/hdfs.h#L259
     hdfs_->hdfsBuilderSetNameNode(builder, "default");
   } else {
-    hdfs_->hdfsBuilderSetNameNode(builder, nn.c_str());
+    hdfs_->hdfsBuilderSetNameNode(builder, nn == "" ? "default", nn.c_str());
```

Comment:
There is a typo here. It should be colon not comma. i.e. `"nn == "" ? "default" : nn.c_str()`.

Reply:
Sorry, I updated the commit again.